pipeline/tests/powdr_std.rs

```diff
@@ -199,3 +205,48 @@ fn ff_inv_big() {
     );
     assert_eq!((result * x) % modulus, 1.into());
 }
+
+#[test]
+fn sort() {
+    let test_inputs = vec![
+        vec![],
+        vec![1],
+        vec![0, 0],
+        vec![1, 2],
+        vec![2, 1],
+        vec![3, 2, 1],
+        vec![0, 0, -1],
+        vec![0, 0, -1, 0, 0, -1, -1, 2],
+        vec![8, 0, 9, 20, 23, 88, 14, -9],
+    ];
+    let code =
+        "let test_sort: int[] -> int[] = |x| std::array::sort(x, |a, b| a < b); machine Main { }"
+            .to_string();
+    let mut pipeline = Pipeline::<GoldilocksField>::default().from_asm_string(code, None);
+    let analyzed = pipeline.compute_analyzed_pil().unwrap().clone();
+    for input in test_inputs {
+        let mut input_sorted = input.clone();
+        input_sorted.sort();
+        let result = evaluate_function(
+            &analyzed,
+            "test_sort",
+            vec![Arc::new(Value::Array(
+                input
+                    .into_iter()
+                    .map(|x| Arc::new(Value::Integer(x.into())))
+                    .collect(),
+            ))],
+        );
+        let Value::Array(result) = result else {
+            panic!("Expected array")
+        };
+        let result: Vec<i32> = result
+            .into_iter()
+            .map(|x| match x.as_ref() {
+                Value::Integer(x) => x.try_into().unwrap(),
+                _ => panic!("Expected integer"),
+            })
+            .collect::<Vec<_>>();
```

Comment:
Turbofish here seems unnecessary, since you explicitly defined the type of result.